kvmagent/kvmagent/test/shareblock_testsuite/test_sharedblock_ha.py

```
@@ -0,0 +1,153 @@
+                    result = {vgUuid: {"lastCheck": linux.get_current_timestamp(), "lastSuccess": linux.get_current_timestamp(), "code": 0, "error": ""}}
+                elif self.zsblk_agent_heart_result == "fail":
+                    result = {vgUuid: {"lastCheck": linux.get_current_timestamp(), "lastSuccess": 1, "code": self.zsblk_agent_heart_code, "error": "heartbeat failed"}}
+                else:
+                    result = {}
+                return jsonobject.dumps(result)
+            return self.origin_json_dump_get(uri, body, headers, fail_soon, print_curl)
+        http.json_dump_get = mock.Mock(side_effect=mock_func)
+
+
+    def mock_management_work(self):
+        def mock_func(uri, body=None, headers={}, fail_soon=False, print_curl=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)